pkg/ctl/install/install.go

```diff
@@ -0,0 +1,435 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/justinbarrick/go-k8s-portforward"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/riywo/loginshell"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	fluxapi "github.com/weaveworks/flux/api/v6"
+	transport "github.com/weaveworks/flux/http"
+	"github.com/weaveworks/flux/http/client"
+	"github.com/weaveworks/flux/install"
+	"github.com/weaveworks/flux/ssh"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
please import this as `apierrors` for consistency, it's how it's we usually import it and how it's usually imported in Kubernetes.

Reply:
Done.